<!-- -->
<launch>
  <arg name="points_topic" default="points_no_ground" />
  <arg name="objects_topic" default="detected_objects" />

  <arg name="use_points" default="true" />
  <arg name="use_objects" default="true" />
  <arg name="vehicle_length" default="4.5" />
  <arg name="vehicle_width" default="1.75" />
  <arg name="resolution" default="0.2" />
  <arg name="cell_width" default="300" />
  <arg name="cell_height" default="150" />
  <arg name="offset_x" default="25.0" />
  <arg name="offset_y" default="0.0" />
  <arg name="offset_z" default="-2.0" />
  <arg name="occupied_cost" default="100" />

	<node pkg="object_map" type="points2costmap" name="points2costmap" output="screen">
    <remap from="points_no_ground" to="$(arg points_topic)" />
    <remap from="detected_objects" to="$(arg objects_topic)" />
    <param name="use_points" value="$(arg use_points)" />
    <param name="use_objects" value="$(arg use_objects)" />
    <param name="vehicle_length" value="$(arg vehicle_length)" />
    <param name="vehicle_width" value="$(arg vehicle_width)" />
    <param name="resolution" value="$(arg resolution)" />
    <param name="cell_width" value="$(arg cell_width)" />
    <param name="cell_height" value="$(arg cell_height)" />
    <param name="offset_x" value="$(arg offset_x)" />
    <param name="offset_y" value="$(arg offset_y)" />
    <param name="offset_z" value="$(arg offset_z)" />
    <param name="occupied_cost" value="$(arg occupied_cost)" />
	</node>

</launch>
